tests/test_panels/test_request_vars.py

```diff
@@ -0,0 +1,225 @@
+# stdlib
+import json
+import os
+from pprint import saferepr
+
+# pypi
+from pyramid.request import Request
+import markupsafe
+
+# local
+from ._utils import _TestDebugtoolbarPanel
+from ._utils import PY2
+
+
+# ------------------------------------------------------------------------------
+
+
+urlencode = None
+if PY2:
+    import urllib
+
+    urlencode = urllib.urlencode
+else:
+    import urllib.parse
+
+    urlencode = urllib.parse.urlencode
```

Comment:
These should be, and might already be, defined in `pyramid_debugtoolbar.compat`.

Reply:
it is, as `url_encode`. fixing.